src/senselab/audio/tasks/voice_cloning/api.py

```diff
@@ -20,48 +21,54 @@ def clone_voices(
         target_audios (List[Audio]): A list of audio samples whose voices will be extracted
             and used to replace the voices in the corresponding source audio samples.
-        model (CoquiTTSModel, optional): The model to use for voice cloning. Currently,
-            all Coqui TTS models are supported. `CoquiTTSModel` is a child class of `SenselabModel`
-            and specifies the model for cloning.
-            If None, the default model "voice_conversion_models/multilingual/multi-dataset/knnvc" is used.
+        model (CoquiTTSModel, optional): The `CoquiTTSModel` model to use for voice cloning.
+            The `CoquiTTSModel` is a child class of `SenselabModel` and specifies the model for cloning.
+            All Coqui TTS models are supported, including "voice_conversion_models/multilingual/multi-dataset/knnvc",
+            "voice_conversion_models/multilingual/vctk/freevc24",
+            "voice_conversion_models/multilingual/multi-dataset/openvoice_v1",
+            "voice_conversion_models/multilingual/multi-dataset/openvoice_v2".
+            If None, the default model is SPARC.
         device (Optional[DeviceType], optional): The preferred device to run the model on (e.g., CPU or GPU).
             Defaults to None.
         **kwargs: Additional keyword arguments for model-specific parameters that will
             be passed directly to the underlying model's voice cloning method.
+            For instance, the default SPARC model accepts a `lang` parameter to specify the language
+            (e.g., `lang=Language("english")`). If `lang` is not None, the model will operate in a
+            multi-language mode.
 
     Returns:
         List[Audio]: A list of audio samples with cloned voices from the corresponding source and target audios.
 
     Raises:
         ValueError: If the lengths of `source_audios` and `target_audios` do not match.
-        NotImplementedError: If the specified model is not supported. Currently, only KNNVC is supported.
+        NotImplementedError: If the specified model is not supported.
+            Currently, only SPARC and Coqui TTS models are supported.
 
     Examples:
         >>> source_audios = [Audio.from_filepath("source1.wav"), Audio.from_filepath("source2.wav")]
         >>> target_audios = [Audio.from_filepath("target1.wav"), Audio.from_filepath("target2.wav")]
         >>> cloned_audios = clone_voices(source_audios, target_audios)
-
-    Todo:
-        Add logging with timestamps.
     """
-    if model is None:
-        model = CoquiTTSModel(path_or_uri="voice_conversion_models/multilingual/multi-dataset/knnvc")
     if len(source_audios) != len(target_audios):
         raise ValueError("The list of source and target audios must have the same length.")
 
-    if isinstance(model, CoquiTTSModel):
+    if model is None:
+        lang = kwargs.pop("lang", None)
+        return SparcVoiceCloner.clone_voices(
```

Comment:
why are we using this as a default for providing none? We should prob have a way to specify a Sparc model similar to how we do with Coqui.